docs/topics/wasm/wasm-js-interop.md

```diff
@@ -1,164 +1,227 @@
 [//]: # (title: Interoperability with JavaScript)
 
-Kotlin/Wasm allows you to both use JavaScript code from Kotlin and Kotlin code from JavaScript.
+Kotlin/Wasm allows you to use both JavaScript code in Kotlin and Kotlin code in JavaScript.
 
-The [Kotlin/JS compiler](js-overview.md) already provides the ability to transpile your Kotlin code to JavaScript. The
-Kotlin/Wasm interoperability with JavaScript is designed in a similar way, taking into account that JavaScript is a
-dynamically typed language compared to Kotlin. Follow our guide to configure interoperability in your projects.
+As with [Kotlin/JS](js-overview.md), the Kotlin/Wasm compiler also has interoperability with JavaScript. If you are 
+familiar with Kotlin/JS interoperability, you will find that Kotlin/Wasm interoperability is similar. However,
+there are key differences to consider. See the comparison between [Kotlin/Wasm and Kotlin/JS](#kotlin-wasm-and-kotlin-js-differences).
 
-Remember that Kotlin/Wasm is in [Alpha](components-stability.md#stability-levels-explained), and some features
-are not supported yet. We're planning to improve interoperability with JavaScript by implementing some of the missing
-features or similar functionality.
+> Kotlin/Wasm is [Alpha](components-stability.md). It may be changed at any time. Use it in scenarios before production. 
+> We would appreciate your feedback on it in [YouTrack](https://kotl.in/issue).
+>
+{type="note"}
+
+## Use JavaScript code in Kotlin
+
+Learn how to use JavaScript code in Kotlin by using `external` declarations, functions with JavaScript code snippets, 
+and the `@JsModule` annotation.
 
-## Use JavaScript code from Kotlin
+### External declarations
 
-### external modifier
+External JavaScript code is not visible in Kotlin by default.
+To use JavaScript code in Kotlin, you can describe its API with `external` declarations.
 
-To access JavaScript declarations defined in the global scope, mark them with the `external` modifier. Consider this
-JavaScript code sample:
+#### JavaScript functions
+
+Consider this JavaScript function: 
 
 ```javascript
-// JavaScript
+function greet(name) {
+    console.log("Hello, " + name + "!");
+}
+```
+
+You can declare it in Kotlin with the `external()` function:
```

Comment:
Not sure that we nee parenthesis in this case. It is not the name of the function but the name of the keyword, am I right?

Reply:
If so, the statement "with the `external()` function" is not correct, it's better to say "declare it in Kotlin as an `external` function:"

Reply:
That's true. The function is greet().